pom.xml

```diff
@@ -20,6 +20,25 @@
     <url>https://www.github.com/Hygieia/</url>
   </organization>
 
+  <repositories>
+    <repository>
```

Comment:
we are already publishing the snapshots to the snapshot repo, can you explain this change.

Reply:
removed